src/Microsoft.IdentityModel.Tokens.Saml/Saml2/Saml2SecurityTokenHandler.cs

```diff
@@ -1129,7 +1134,7 @@ protected virtual void ProcessStatements(ICollection<Saml2Statement> statements,
                 else if (statement is Saml2AuthorizationDecisionStatement authzStatement)
                     ProcessAuthorizationDecisionStatement(authzStatement, identity, issuer);
                 else
-                    LogWarning(LogMessages.IDX13516, this.GetType());
+                    LogWarning(LogMessages.IDX13516, LogHelper.MarkAsNonPII(this.GetType()));
```

Comment:
shouldn't this use _className?

Reply:
The IDX13516 message is

<https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/fc59ad574ee0e0b9026c405fb566f343bc314bd7/src/Microsoft.IdentityModel.Tokens.Saml/Saml2/LogMessages.cs#L52>

so this should actually be `statement.GetType()` rather than `this.GetType()`.

Reply:
You're right @KalleOlaviNiemitalo. Updated to statement.GetType()